cmd/compose/compose.go

```diff
@@ -277,7 +278,12 @@ const PluginName = "compose"
 
 // RunningAsStandalone detects when running as a standalone program
 func RunningAsStandalone() bool {
```

Comment:
Should have been replaced by https://github.com/docker/cli/blob/ae41fd913eea86eb4be153782846c33935a1039e/cli-plugins/plugin/plugin.go#L223

Reply:
Agree! 😅